Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/AppInsightsClient.java

```diff
@@ -25,13 +25,20 @@
 import com.microsoft.applicationinsights.TelemetryClient;
 import com.microsoft.azuretools.adauth.StringUtils;
 import com.microsoft.azuretools.azurecommons.helpers.Nullable;
-
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.UUID;
 
 public class AppInsightsClient {
+    private static final String EVENT_SPLIT = "/";
+    private static final String OPERATION_ID = "operationid";
```

Comment:
keep consistency in prop names, i.e. keep using camel case.

Reply:
although the keys get transformed to the lower case, it's better to keep the original data consistency, and for better readability. (see your own screenshot)

Reply:
agree, done